lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -3005,6 +3005,7 @@ MHWRender::MShaderInstance* HdVP2Material::CompiledNetwork::_CreateMaterialXShad
         }
 
         shaderInstance = shaderMgr->getFragmentShader(fragmentName, "outColor", true);
+        shaderInstance->addInputFragment("NwFaceCameraIfNAN", "output", "Nw");
```

Comment:
This line is MaterialX-specific, UsdPreviewSurface fragments already reference the `NwFaceCameraIfNAN` fragment in their XML.